tests/test-ses.js

```diff
@@ -0,0 +1,70 @@
+"use strict";
+
+const fs = require("fs");
+const httpMocks = require("node-mocks-http");
+const path = require("path");
+const test = require("tape-async");
+
+const DB = require("../db/DB");
+const getSha1 = require("../sha1-utils");
+const ses = require("../controllers/ses");
+
+
+const createRequestBody = function(notificationType) {
+  const cwd = path.join(path.dirname(fs.realpathSync(__filename)));
```

Comment:
Wow. Not sure if there is an easier way to get the cwd.
I'd have to do some researching into the differences between this, using `__dirname`, versus [`process.cwd()`](https://nodejs.org/api/process.html#process_process_cwd).

https://stackoverflow.com/questions/9874382/whats-the-difference-between-process-cwd-vs-dirname

Reply:
`__dirname` works for me.